### PR TITLE
Prepare for new version

### PR DESCRIPTION
**Pull request by zuiderkwast**
* Restore backward compatibility with wooga/eredis (API functions
  and default reconnect timeout)
* Update CHANGELOG.md and README.md
* Update version in app.src file

### CODE REVIEW

**Comment by bjosv** [comment]
Seems there is an AUTHORS file aswell

**Review by bjosv — APPROVED** [review]

**Comment by bjosv** [comment]
Docs needs to be re-added though

**Comment by zuiderkwast** [comment]
> Docs needs to be re-added though

I'll rebase this one and update some stuff, add link to docs from README, etc.

File: README.md [hunk 1]
```diff
@@ -133,15 +125,15 @@ Redis, without any type conversion. If Redis returns a list of values,
...
+### Pipelined query: [qp/2,3](doc/eredis.md#qp-2)
 
 To send multiple requests to redis in a batch, aka. pipelining
 requests, you may use `eredis:qp(Client::pid(),
 [Command::iolist()])`. This function returns `{ok, [Value::binary()]}`
 where the values are the redis responses in the same order as the
 commands you provided.
 
-### Connect a client: start_link/0-3
+### Connect a client: [start_link/0,1,2,3](doc/eredis.md#start_link-0)
```
Comment (bjosv) [inline README.md:136]: 0 - ~7 now?

**Review by bjosv — APPROVED** [review]
LGTM

**Pull request merged**